examples/mobile/ScheduleApp/js/app.js

```diff
@@ -178,11 +199,19 @@
 			case "vclick":
 				self._onClick(event);
 				break;
+			case "tizenhwkey":
+				self._onHWKey(event);
+				break;
+			case "tauback":
+				self._onBack(event);
+				break;
 		}
 	};
 
 	prototype.init = function () {
 		document.addEventListener("pagebeforeshow", this, true);
+		window.addEventListener("tizenhwkey", this, false);
+		document.addEventListener("tauback", this, false);
```

Comment:
Why are there two features to go back?
Is it a different function?

Reply:
The `tizenhwkey` is not the same as `back` and this event not exists on android os

> (2) Use ‘navigate up (back key)’ button to ‘Reset’ the application.
> Is it for easy testing?

Yes. This is for testing. This is feature from issue: https://github.com/Samsung/TAU/issues/1181